scripts/ci/_nx-affected-targets.sh

```diff
@@ -12,7 +12,7 @@ export BASE=${BASE:-main}
 AFFECTED_ALL=${AFFECTED_ALL:-} # Could be used for forcing all projects to be affected (set or create `secret` in GitHub with the name of this variable set to the name of the branch that should be affected, prefixed with the magic string `7913-`)
 BRANCH=${BRANCH:-$GITHUB_HEAD_REF}
 
-if [[ -n "$BRANCH" && -n "$AFFECTED_ALL" && "$AFFECTED_ALL" == "7913-$BRANCH" ]]; then
+if [[ (-n "$BRANCH" && -n "$AFFECTED_ALL" && "$AFFECTED_ALL" == "7913-$BRANCH") || (-n "$NX_AFFECTED_ALL" && "$NX_AFFECTED_ALL" == "true") ]]; then
```

Comment:
You don't need to check whether the variable is non-empty first, it's implicit in the second check. Also, @lodmfjord mentioned `unbound variable` error.
```suggestion
: "${NX_AFFECTED_ALL:=false}"

if [[ (-n "$BRANCH" && -n "$AFFECTED_ALL" && "$AFFECTED_ALL" == "7913-$BRANCH") ||"$NX_AFFECTED_ALL" == "true" ]]; then
```